v7/config_fetcher.go

```diff
@@ -129,7 +129,7 @@ func (f *configFetcher) runPoller(pollInterval time.Duration) {
 			return
 		}
 		if err := f.refreshIfOlder(f.ctx, time.Now().Add(-pollInterval/2), true); err != nil {
-			f.logger.Errorf("cannot refresh configcat configuration: %v", err)
+			f.logger.Errorf(0, "cannot refresh configcat configuration: %v", err)
```

Comment:
This SDK uses a bunch of log messages which are not present in other SDKs. I haven't introduce event ids for these. (Which is probably OK.) However, these messages is mostly lowercased and lacks dot at the end, which we should probably make consistent with the rest of the messsages.